proprietary/purmerend-design-tokens/figma/figma.tokens.json

```diff
@@ -1952,9 +1959,13 @@
           "$type": "borderWidth",
           "$value": "{basis.border-width.sm}"
         },
+        "border-color": {
+          "$type": "color",
+          "$value": "{basis.color.secondary-inverse.border-4}"
+        },
```

Comment:
Je haalt nu voor één component tokens uit drie verschillende kleurschalen, primary-inverse en secondary-inverse én text; dit lijkt me niet de logische aanpak. In principe moeten de tokens uit één kleurschaal komen.

Zou je er op willen letten bij andere componenten dat we hetzelfde principe volgen?